sprint/start_clean.cmd

```diff
@@ -22,6 +22,9 @@ ECHO #
 ECHO ####
```

Comment:
maintaining cmd scripts is quite a burden, and it's a horrible scripting language to boot. My strategy I think is to install/require git-bash for all windows users, and use bash scripts that are identical for the different operating systems.

Reply:
I think initially Brian was looking babun (and docksal). Babun would be nice, but increases package size by a bunch. Git Bash is a bit limited, but decent enough though requires more extra steps for Windows users. Powershell would be nice too, despite the additional burden, but not bundled with Windows 7 by default (not sure about 10 Home).

That written, :+1: to consolidation.